src/libecalc/fixtures/cases/ltp_export/ltp_power_from_shore_yaml.py

```diff
@@ -0,0 +1,92 @@
+from pathlib import Path
+
+import yaml
+
+from libecalc.common.time_utils import Frequency
+from libecalc.dto import Asset, ResultOptions, VariablesMap
+from libecalc.expression.expression import ExpressionType
+from libecalc.fixtures.cases.ltp_export import data
+from libecalc.presentation.yaml.mappers.variables_mapper import map_yaml_to_variables
+from libecalc.presentation.yaml.model import PyYamlYamlModel, YamlModel
+from libecalc.presentation.yaml.parse_input import map_yaml_to_dto
+
+
+def ltp_pfs_yaml_factory(
+    regularity: float,
+    cable_loss: ExpressionType,
+    max_usage_from_shore: ExpressionType,
+    load_direct_consumer: float,
+) -> [Asset, VariablesMap]:
+    input_text = f"""
+    START: 2025-01-01
```

Comment:
Can we add this yaml case as a fixture in conftest? Specifically in `valid_example_case_yaml_case`, as this will make sure we test that the model is valid with the yaml validation. I think this PR is missing the Yaml class update, similar to the update in dto.

Reply:
Have tried to add fixture in conftest. The Yaml class is updated.